cookbooks/aws-parallelcluster-slurm/templates/default/slurm/resume_program.erb

```diff
@@ -1,2 +1,13 @@
 #!/bin/bash
-sudo -u <%= node['cluster']['cluster_admin_user'] %> <%= node_virtualenv_path %>/bin/slurm_resume "$@"
+# ResumeProgram should read SLURM_RESUME_FILE within ten seconds of starting to guarantee that it still exists.
```

Comment:
Is this timeout configurable? Wouldn't it make sense to extend this timeout to avoid scheduling issues?

Reply:
no, it's not configurable. This is a reason why the copy it's done as first thing in the resume program execution